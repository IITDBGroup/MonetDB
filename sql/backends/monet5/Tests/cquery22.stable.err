stderr of test 'cquery22` in directory 'sql/backends/monet5` itself:


# 15:42:42 >  
# 15:42:42 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34536" "--set" "mapi_usock=/var/tmp/mtest-14134/.s.monetdb.34536" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-trails/BUILD/var/MonetDB/mTests_sql_backends_monet5" "--set" "embedded_r=yes" "--set" "embedded_py=true"
# 15:42:42 >  

# builtin opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-trails/BUILD/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 34536
# cmdline opt 	mapi_usock = /var/tmp/mtest-14134/.s.monetdb.34536
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-trails/BUILD/var/MonetDB/mTests_sql_backends_monet5
# cmdline opt 	embedded_r = yes
# cmdline opt 	embedded_py = true
# cmdline opt 	gdk_debug = 553648138

# 15:42:42 >  
# 15:42:42 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-14134" "--port=34536"
# 15:42:42 >  

MAPI  = (monetdb) /var/tmp/mtest-14134/.s.monetdb.34536
QUERY = start continuous function calculateaverage() with cycles 2 as calcavg; --error
ERROR = !The continuous function calcavg is already registered.
CODE  = 3F000
MAPI  = (monetdb) /var/tmp/mtest-14134/.s.monetdb.34536
QUERY = start continuous function calculateaverage(12) with cycles 4 as calcavg; --error
ERROR = !The continuous function calcavg is already registered.
CODE  = 3F000
MAPI  = (monetdb) /var/tmp/mtest-14134/.s.monetdb.34536
QUERY = start continuous function calculateaverage(10) as calc; --error
ERROR = !The continuous function calc is already registered.
CODE  = 3F000
MAPI  = (monetdb) /var/tmp/mtest-14134/.s.monetdb.34536
QUERY = start continuous function calculateaverage() as othercalc; --error
ERROR = !The continuous function othercalc is already registered.
CODE  = 3F000

# 15:42:44 >  
# 15:42:44 >  "Done."
# 15:42:44 >  

