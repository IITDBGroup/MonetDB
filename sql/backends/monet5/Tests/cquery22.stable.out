stdout of test 'cquery22` in directory 'sql/backends/monet5` itself:


# 15:42:42 >  
# 15:42:42 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34536" "--set" "mapi_usock=/var/tmp/mtest-14134/.s.monetdb.34536" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-trails/BUILD/var/MonetDB/mTests_sql_backends_monet5" "--set" "embedded_r=yes" "--set" "embedded_py=true"
# 15:42:42 >  

# MonetDB 5 server v11.28.0
# This is an unreleased version
# Serving database 'mTests_sql_backends_monet5', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 128bit integers
# Found 15.496 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2018 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.localdomain:34536/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-14134/.s.monetdb.34536
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
# MonetDB/Timetrails module loaded
# MonetDB/Python2 module loaded
# MonetDB/R   module loaded

Ready.
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_querylog.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_temporal.sql
# loading sql script: 18_index.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 25_debug.sql
# loading sql script: 26_sysmon.sql
# loading sql script: 27_rejects.sql
# loading sql script: 39_analytics.sql
# loading sql script: 39_analytics_hge.sql
# loading sql script: 40_geom.sql
# loading sql script: 40_json.sql
# loading sql script: 40_json_hge.sql
# loading sql script: 41_md5sum.sql
# loading sql script: 45_uuid.sql
# loading sql script: 46_profiler.sql
# loading sql script: 50_cquery.sql
# loading sql script: 51_sys_schema_extension.sql
# loading sql script: 60_wlcr.sql
# loading sql script: 72_fits.sql
# loading sql script: 74_netcdf.sql
# loading sql script: 75_lidar.sql
# loading sql script: 75_shp.sql
# loading sql script: 75_storagemodel.sql
# loading sql script: 80_statistics.sql
# loading sql script: 80_udf.sql
# loading sql script: 80_udf_hge.sql
# loading sql script: 85_bam.sql
# loading sql script: 90_generator.sql
# loading sql script: 90_generator_hge.sql
# loading sql script: 97_comments.sql
# loading sql script: 99_system.sql

# 15:42:42 >  
# 15:42:42 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-14134" "--port=34536"
# 15:42:42 >  

#create stream table cqinput22(input integer) set window 2;
#create function calculateaverage() returns real begin return select AVG(input) from cqinput22; end;
#create function calculateaverage(dev int) returns real begin return select AVG(dev + input) from cqinput22; end;
#start continuous function calculateaverage() with cycles 3 as calcavg;
#start continuous function calculateaverage() with cycles 1 as calc;
#start continuous function calculateaverage(9) with cycles 1 as othercalc;
#stop all continuous;
#drop function calculateaverage();
#drop function calculateaverage(int);
#drop table cqinput22;

# 15:42:44 >  
# 15:42:44 >  "Done."
# 15:42:44 >  

